extensions/amp-viewer-integration/0.1/messaging/messaging.js

```diff
@@ -14,10 +14,9 @@
  * limitations under the License.
  */
 
-import {getData} from '../../../../src/event-helper';
-import {parseJson} from '../../../../src/json';
-
 const TAG = 'amp-viewer-messaging';
+const CHANNEL_OPEN_MSG = 'channelOpen';
+const HANDSHAKE_POLL_MSG = 'handshake-poll';
 export const APP = '__AMPHTML__';
 
 /**
```

Comment:
Could you pls try removing `export` from `Message`, `RequestHandler`, and `MessageType`?

Reply:
Done